easybuild/easyconfigs/s/S4/S4-1.1.1-7fd00a2-foss-2017b.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'MakeCp'
+
+name = 'S4'
+commit = '7fd00a2'
+version = '1.1.1-%s' % commit
```

Comment:
normally, when using a specific commit, the commit date is used for the version.